README.md

```diff
@@ -35,10 +35,12 @@ npm install stereo-img
   - `right-left`: left eye on the right, right eye on the left, Exif angle of view is used if present.
   - `top-bottom`: left eye on the top, right eye on the bottom, Exif angle of view is used if present.
   - `bottom-top`: left eye on the bottom, right eye on the top, Exif angle of view is used if present.
+  - `pair`: separate files for left and right image. Left eye image read from src attribute, right eye image read from attribute rightsrc.
```

Comment:
I suggest using `src-right` as attribute, this will make it start with `src`and therefore next to it in case attributes get sorted and also more obvious this is an `src`-like attribute